### PR TITLE
Support Ipv6 in builder services

### PR DESCRIPTION
**Pull request by reset**
Signed-off-by: Jamie Winsor <jamie@vialstudios.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by reset** [comment]
@adamhjk this small change extends all the builder services to support ipv6 like we did in the supervisor's http gateway

File: components/net/src/routing.rs [hunk 1]
```diff
@@ -203,7 +202,7 @@ impl Broker {
     /// # Panics
     ///
     /// * Broker crashed during startup
-    pub fn run(net_ident: String, routers: &Vec<net::SocketAddrV4>) -> JoinHandle<()> {
+    pub fn run<T: ToAddrString>(net_ident: String, routers: &Vec<T>) -> JoinHandle<()> {
```
Comment (adamhjk) [inline components/net/src/routing.rs]: Oh I like this quite a bit.

**Comment by adamhjk** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by adamhjk** [comment]
image

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/180530436).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/180530436).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**